content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -31,6 +31,8 @@ A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is simila
 
 `VolumeSnapshotClass` allows you to specify different attributes belonging to a `VolumeSnapshot`. These attributes may differ among snapshots taken from the same volume on the storage system and therefore cannot be expressed by using the same `StorageClass` of a `PersistentVolumeClaim`.
 
+Volume snapshots provide Kubernetes users with a standardized way to copy a volume's contents at a particular point in time without creating an entirely new volume. This functionality enables, for example, database administrators to backup databases before performing edit or delete modifications.
```

Comment:
(nit):
```suggestion
Volume snapshots provide you with a standardized way to copy a volume's contents at a particular point in time, without creating an entirely new volume. This functionality enables, for example, database administrators to back up the storage for a database before performing edit or delete modifications.
```

> edit or delete modifications
You wouldn't usually snapshot the storage before running `UPDATE` or `DELETE`. `DROP TABLE`, you might. Is there wording that can highlight the second case as being a good fit?

If I have made a snapshot and now regret running that `DROP TABLE`, what do my next steps look like? How do I take the `VolumeSnapshot` and turn that into an outcome that the end users are happy with? (this could be something we leave for explaining in a separate GitHub issue).